crates/fraud_proofs/global_merkle_root/storage/src/update.rs

```diff
@@ -138,11 +142,33 @@ where
         // TODO(#2584): Insert state transition bytecode and consensus parameter updates.
         // TODO(#2585): Insert uplodade bytecodes.
         // TODO(#2586): Insert blobs.
-        // TODO(#2587): Insert raw code for created contracts.
+        if let Transaction::Create(tx) = tx {
+            self.process_create_transaction(tx)?;
+        }
 
         Ok(())
     }
 
+    // CreateBody is not publicly exported, so we need to use a generic type
+    // and ensure that all trait bounds are satisfied
+    fn process_create_transaction(&mut self, tx: &Create) -> anyhow::Result<()> {
+        let bytecode_witness_index = tx.bytecode_witness_index();
+        let witnesses = tx.witnesses();
+        let bytecode = witnesses[(*bytecode_witness_index) as usize].as_vec();
```

Comment:
If this is infallible, I'd suggest:
```suggestion
        let bytecode = witnesses[usize::from(*bytecode_witness_index)].as_vec();
```

Else I'd suggest using `try_from` and return the error.

Reply:
it is infallible, will commit your suggestion